internal/backend/rclone/backend.go

```diff
@@ -73,6 +73,13 @@ func run(command string, args ...string) (*StdioConn, *exec.Cmd, *sync.WaitGroup
 	if err != nil {
 		return nil, nil, nil, nil, err
 	}
+	// close rclone side of pipes
+	if err := r.Close(); err != nil {
+		return nil, nil, nil, nil, err
```

Comment:
In this case, it looks like w is never closed.

Reply:
I have to admit I was too lazy to improve the resource cleanup here after an error. I'm still not sure I've caught everything.